src/connections/DashboardEntity.ts

```diff
@@ -0,0 +1,24 @@
+import { v3Resource } from "@/connections/util/apiConnectionFactory";
+import { connectionHook } from "@/connections/util/connectionShortcuts";
+import {
+  dashboardEntityIndex,
+  DashboardEntityIndexQueryParams,
+  DashboardEntityIndexResponse
+} from "@/generated/v3/dashboardService/dashboardServiceComponents";
+import ApiSlice from "@/store/apiSlice";
+import { Filter } from "@/types/connection";
+
+type ExtendedDashboardQueryParams = DashboardEntityIndexQueryParams & {
+  ["page[size]"]?: number;
+  ["page[number]"]?: number;
```

Comment:
If these aren't getting generated automatically, that's a problem in the BE documentation.